src/sentry/features/manager.py

```diff
@@ -66,8 +68,37 @@ def has(self, name, *args, **kwargs):
 
         >>> FeatureManager.has('organizations:feature', organization, actor=request.user)
         """
+        return self.build()(name, *args, **kwargs)
+
+    def build(self):
```

Comment:
I wish this were a three-letter verb. 😭

More seriously, `build` is a placeholder name that explains nothing. I would love suggestions to clarify what this method is returning. Or the name `FeaturePredicate` for that matter.

As best as I can explain, `FeaturePredicate` is identical in function to the `FeatureManager.has` method, except that it is scoped to a single snapshot in time. (Perhaps confusingly, the "snapshot in time" refers only to the behavior of the `plugins.all` function -- which, to reiterate, I'm assuming can change at runtime but it might actually not. Changes in state due to `FeatureManager.add` actually *would* be reflected by the returned `FeaturePredicate`, although I'd rather not have code depend on that either.)